packages/web-react/src/constants/dictionaries.ts

```diff
@@ -5,6 +5,12 @@ export const AlignmentX = {
   RIGHT: 'right',
 } as const;
 
+export const AlignmentY = {
```

Comment:
Can we please use this constant instead of values in all the places where value is used? At least in internal implementation. In demos, there can be values. But there are multiple times repeated `top`, `center`, and `bottom`. Maybe the `center` the most as it is a default value for most cases.

Reply:
Good point, thank you!